client.py

```diff
@@ -32,8 +35,8 @@
 
 
 class Client:
-    def __init__(self, loop, my_id, server, ssid='', pw='',
-                 port=8123, timeout=2000,
+    def __init__(self, loop, my_id, server, port=8123,
```

Comment:
I'm sorry but I see no logic in dividing server and port postition. These belong together and therefore should be put together.